backend/routers/transcribe_v2.py

```
@@ -20,7 +21,7 @@
 from utils.stt.streaming import *
 from utils.webhooks import send_audio_bytes_developer_webhook, realtime_transcript_webhook, \
     get_audio_bytes_webhook_seconds
-from utils.pusher import connect_to_transcript_pusher, connect_to_audio_bytes_pusher
+from utils.pusher import connect_to_transcript_pusher, connect_to_audio_bytes_pusher, connect_to_trigger_pusher
```

Ruff

`utils.pusher.connect_to_transcript_pusher` imported but unused

Remove unused import

(F401)